lib/averages.js

```diff
@@ -0,0 +1,43 @@
+function total(array){
+  let sum = 0;
+  array.forEach(function(value){
+  	sum = sum + value;
+  });
+  return sum;
+}
+let array = [1,2,3,4,5,3];
+console.log(total(array));
+
+function mean(array){
+  return (total(array)/array.length);
+}
+console.log(mean(array));
+
+function median(array){
+  array.sort(function(a, b){
```

Comment:
could we pull this out into a separate function since this is pretty independent (but necessary) to finding the median?